public-html/aboutus.php

```diff
@@ -2,15 +2,35 @@
 require $_SERVER['DOCUMENT_ROOT'] . '/../vendor/autoload.php';
 use src\SessionCookie;
 $session = new SessionCookie;
-$session->includeAccess();
 ?>
 <html>
 <title>About us</title>
 
 <head>
+    <?php
+    session_start();
+    require_once($_SERVER["DOCUMENT_ROOT"] . "/../vendor/autoload.php");
```

Comment:
Check if these two lines are needed, since they seems already be covered at top and session constructor